specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/examples/ManagedInstanceCreateMax.json

```diff
@@ -44,7 +46,8 @@
           "fullyQualifiedDomainName" : "testinstance.database.windows.net",
           "administratorLogin" : "dummylogin",
           "subnetId" : "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Network/virtualNetworks/vnet1/subnets/subnet1",
-          "state" : "Ready"
+          "state": "Ready",
+          "dnsZone":  "1b4e2caff2530"
```

Comment:
Is this a realistic-looking dns zone?

Reply:
Yes it is. We generate dns zone with the first part of a new Guid.